### PR TITLE
Fix cosmetics

### PR DESCRIPTION
**Pull request by d4straub**
Fixes some typo's, the figure docs/images/ampliseq_workflow_multiplesequencingruns.png/.svg, and improves docs/usage.md layout.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Fixed`
 
 - [#428](https://github.com/nf-core/ampliseq/pull/428) - Fixed samplesheet sampleID entries, now allows dashes.
+- [#433](https://github.com/nf-core/ampliseq/pull/433) - Fixed typo's and improved documentation layout.
```
Comment (erikrikarddaniel) [inline CHANGELOG.md:17]: There's a typo here: "typo's" should be "typos" as it's plural, not genitive! :-)

**Review by erikrikarddaniel — APPROVED** [review]
Except for a type, looks fine.

File: CHANGELOG.md (hunk 1 above)
Comment (d4straub) [inline CHANGELOG.md:17]:
```suggestion
- [#433](https://github.com/nf-core/ampliseq/pull/433) - Fixed typos and improved documentation layout.
```

Comment (d4straub) [reply CHANGELOG.md:17]: True, thanks!

**Pull request merged**